### PR TITLE
feat: add your first chart & update installation

### LINKED ISSUE #46: 💡 Update Docs - Update Introduction - Installation - Your first chart
## Summary

We need to update the documentation and add the following sections : 
* Introduction
* Installation
* Your first chart

### PR DESCRIPTION
**Pull request by nour-borgi**
# Motivation

- Added "Your first chart" section in the documentation.
- Updated "Installation" section in the documentation.

Fixes #46 

## Type of change

Please delete options that are not relevant.

- [x] Documentation

### CODE REVIEW

File: packages/ez-react/src/docs/3_yourfirstchart.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+EazyChart provides a set of ready-to-use charts that you can fully customize. In the example below, you will be adding a "Bar Chart" to a React project. First, you need to import the **BarChart** component from the library.
+
+```
+# Code for importing the bar chart component
+Import { BarChart } from 'eazychart-react'
+Import 'eazychart-css'
+```
+
+The above code will import the Bar chart to where you want it to show.
```
Comment (marrouchi) [inline packages/ez-react/src/docs/3_yourfirstchart.stories.mdx:17]:
```suggestion
EazyChart provides a set of ready-to-use charts that you can fully customize. In the example below, you will be adding a "Bar Chart" to a React project. First, you need to import the **BarChart** component from the library.

```js
// Code for importing the bar chart component
Import { BarChart } from 'eazychart-react'
Import 'eazychart-css'
```

The above code will import the Bar chart to where you want it to show. 
```

File: packages/ez-react/src/docs/3_yourfirstchart.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+Hang on there, you still have a couple of things to do. You need to define the data that the chart should display: 
+
+```
+Const data=[
+    { country: 'Italy', population: 65 },
+    { country: 'Spain', population: 46 },
+    { country: 'Australia', population: 25 },
+];
+```
+Awesome. Now you’ll need to configure the chart. To do so, we will need to create an object that will contain all the props we need to define.
```
Comment (marrouchi) [inline packages/ez-react/src/docs/3_yourfirstchart.stories.mdx:36]:
```suggestion
Hang on there, you still have a couple of things to do. You need to define the data that the chart should display: 

```js
const data=[
    { country: 'Italy', population: 65 },
    { country: 'Spain', population: 46 },
    { country: 'Australia', population: 25 },
];
```
Awesome. Now you’ll need to configure the chart. To do so, we will need to create an object that will contain all the props we need to define.
```

File: packages/ez-react/src/docs/3_yourfirstchart.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+};
+```
+
+Now that everything is in order, run your project using npm or yarn, open up your browser, and you should be able to see your chart on your screen. 
+
+`Npm run start`
+
+Or 
+
+`Yarn start`
```
Comment (marrouchi) [inline packages/ez-react/src/docs/3_yourfirstchart.stories.mdx:109]: Let's lowercase all the commands .. :)

**Review by marrouchi — CHANGES_REQUESTED** [review]

File: packages/ez-react/src/docs/2_installation.stories.mdx [hunk 4]
```diff
@@ -4,14 +4,14 @@ import { Meta } from '@storybook/addon-docs/blocks';
 
 # Installation
 
-First things first, you need to install the EazyChart library for your project. EazyChart is available in both React and Vue web apps. So open up your terminal of choice, navigate to your project directory and use one of the commands below to install the library :
+First things first, you need to install the **EazyChart** library for your project. EazyChart is available in both **React** and **Vue** web apps. So open up your terminal of choice, navigate to your project directory and use one of the commands below to install the library :
 
 If you are using npm for your project, use one of the command line below depending on the project:
 ```
```
Comment (marrouchi) [inline packages/ez-react/src/docs/2_installation.stories.mdx:10]:
```suggestion
```sh
```

File: packages/ez-react/src/docs/2_installation.stories.mdx [hunk 5]
```diff
@@ -4,14 +4,14 @@ import { Meta } from '@storybook/addon-docs/blocks';
...
-npm i --save eazychart-react
+npm i --save eazychart-react eazychart-css
 ```
 If you are using yarn for your project, use one of the command line below depending on the project:
 ```
```
Comment (marrouchi) [inline packages/ez-react/src/docs/2_installation.stories.mdx:14]:
```suggestion
```sh
```

File: packages/ez-react/src/docs/3_yourfirstchart.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+![Example of Bar Chart](gif)
+
+You can learn more about more properties in the Axis section.
+
+Cool, now that everything is set up in place, let’s write the JSX code to display the chart using the props that we defined earlier.  
+
+Your code should look like this: 
+
+```jsx
```
Comment (marrouchi) [inline packages/ez-react/src/docs/3_yourfirstchart.stories.mdx:70]:
```suggestion
```react
```

File: packages/ez-react/src/docs/3_yourfirstchart.stories.mdx [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+  },
+  data,
+};
+
+return (
+  <div>
+    <BarChart {...props} />
+  </div>
+)};
+```
```
Comment (marrouchi) [inline packages/ez-react/src/docs/3_yourfirstchart.stories.mdx:100]: Let's prettier this :D

File: packages/ez-react/src/docs/3_yourfirstchart.stories.mdx [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+> **NOTE**  `!` 
+> **You can import it once into your app main file, and it won't be systematic with every chart import.**
+
+Cool, now that you imported the Bar chart from the library, you are ready to use it in your project. 
+
+Hang on there, you still have a couple of things to do. You need to define the data that the chart should display: 
+
+```js
+const data=[
```
Comment (marrouchi) [inline packages/ez-react/src/docs/3_yourfirstchart.stories.mdx:30]:
```suggestion
const data = [
```

**Review by marrouchi — CHANGES_REQUESTED** [review]

File: packages/ez-react/src/docs/3_yourfirstchart.stories.mdx (hunk 6 above)
Comment (nour-borgi) [reply packages/ez-react/src/docs/3_yourfirstchart.stories.mdx:70]: I think react doesn't work
this is the output when I put jsx: 
image

And this is the output when I put react: 
image

File: packages/ez-react/src/docs/3_yourfirstchart.stories.mdx (hunk 7 above)
Comment (nour-borgi) [reply packages/ez-react/src/docs/3_yourfirstchart.stories.mdx:100]: cool:)

File: packages/ez-react/src/docs/2_installation.stories.mdx (hunk 4 above)
Comment (nour-borgi) [reply packages/ez-react/src/docs/2_installation.stories.mdx:10]: :+1:

File: packages/ez-react/src/docs/2_installation.stories.mdx (hunk 5 above)
Comment (nour-borgi) [reply packages/ez-react/src/docs/2_installation.stories.mdx:14]: :+1:

File: packages/ez-react/src/docs/3_yourfirstchart.stories.mdx [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+# Your First Chart
+
+Now that you installed **EazyChart**, you will have the ability to use it and add a chart in your **React** or **Vue** web project. 
+
+EazyChart provides a set of ready-to-use charts that you can fully customize. In the example below, you will be adding a "Bar Chart" to a React project. First, you need to import the **BarChart** component from the library.
+
+```js
+// Code for importing the bar chart component
+import { BarChart } from 'eazychart-react'
+import 'eazychart-css'
```
[the remark this answers is not in the document]
Comment (nour-borgi) [reply packages/ez-react/src/docs/3_yourfirstchart.stories.mdx:14]: :+1:

File: packages/ez-react/src/docs/3_yourfirstchart.stories.mdx (hunk 8 above)
Comment (nour-borgi) [reply packages/ez-react/src/docs/3_yourfirstchart.stories.mdx:30]: :+1:

**Review by marrouchi — APPROVED** [review]
:rocket:  :partying_face: ! Awesome job :)

**Pull request merged**